doc/user/ngctl_guide.md

```diff
@@ -0,0 +1,89 @@
+
+# Overview
+ngctl is a terminal cmd tool for nebula-operator, it has the following commands:
+- [ngctl use](#ngctl-use)
+- [ngctl console](#ngctl-console)
+- [ngctl list](#ngctl-list)
+- [ngctl check](#ngctl-check)
+- [ngctl info](#ngctl-info)
+
+## ngctl use
+`ngctl use` specify a nebula cluster to use. By using a certain cluster, you may omit --nebulacluster option in many control commands.
+
+```
+Examples:
+  # specify a nebula cluster to use
+  ngctl use demo-cluster
+
+  # specify kubernetes context and namespace
```

Comment:
I thanks this is just specify `namespace`, and don't support specify kubernetes context.